composer.json

```diff
@@ -19,6 +19,7 @@
     },
     "require": {
         "php": ">=5.5.0",
+        "composer/semver": "^1|^3.2",
```

Comment:
Do we need to support composer/semver:^1? I don't think that we do, at this point. If we do, we should have tests. I think it's fine to just drop the constraint and only use / test ^3.2.

Reply:
Fixed